src/statsig/constants.ts

```diff
@@ -111,6 +111,20 @@ export const DynamicConfigs = {
     configName: StatsigDynamicConfigs.APP_CONFIG,
     defaultValues: {
       minRequiredVersion: '0.0.0',
+      externalLinks: {
+        baseUrl: 'https://valora.xyz/',
```

Comment:
nit: Maybe? so it's more like the other links?
```suggestion
        web: 'https://valora.xyz',
```

But I guess we probably use it to construct other links so maybe `baseUrl` is fine? 😄

Reply:
yeah `web` is good, we do use it to construct other links but `web` is perhaps easier to understand as a non engineer